rootfs/bin/is_running

```diff
@@ -4,4 +4,4 @@
 set -e
 
 # check if database is running
-gosu postgres pg_ctl status
+gosu postgres psql -l -t >/dev/null 2>&1
```

Comment:
I see. Is this pseudo code acceptable, then?

```
if [[ -f "$PGDATA/recovery.conf" ]]; then
  # We don't need to call psql for checking in the recovery mode.
  # As WAL-E calls psql internally.
  gosu postgres pg_ctl status
else
  gosu postgres pqsl -l -t >/dev/null 2>&1
fi
```

I agree your patch to `is_running` is better in the normal mode.
As sometimes it can be occurred "postmsater process is living but don't reply any queries."  

But basically spawing psql process is slightly heavy.
IMO, it should be lightweight as possible as it can especially in the recovery mode.